### DISCUSSION TITLE
New provider changes duplicate launch.json options

### DISCUSSION THREAD

**Discussion by angles-n-daemons** [mfussenegger/nvim-dap#1263] [Q&A] [answered]
I'm not sure if this is expected, but my nvim-dap configuration has recently started duplicating my launch.json options. This means if I have two configurations in my launch.json file, nvim-dap will present four options for `all_configs`.

I've narrowed it down to the call to load_launchjs from within my own nvim-dap setup file. Is this expected behavior?

**Comment by mfussenegger** [comment, marked as answer]
This is due to https://github.com/mfussenegger/nvim-dap/pull/1237 and expected. You should be able to get rid of your own `load_launchjs` call.

**Comment by angles-n-daemons** [comment]
Awesome, thanks for the response and the great work 🙏
